.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+### Issue details
+
+_Please provide issue details here_.
+
+### Steps to reproduce/test case
+
+_Please provide necessary steps for reproduction of this issue, or better the
+reduced test case (without any external dependencies)_.
+
+### Affected node.js versions
+
+- [ ] master
+- [ ] v5.x
+- [ ] v4.x
+- [ ] v0.12
+- [ ] v0.10
+
+### Affected platforms
+
+- [ ] linux
```

Comment:
Nit: For this and the others, we probably want to style as the project/vendor/whoever styles it. So, `linux`->`Linux`, `windows`->`Windows`, `freebsd`->`FreeBSD`, `solaris`->`Solaris`.